src/modules/jobPost/jobPost.service.ts

```diff
@@ -40,6 +40,21 @@ export class JobPostService {
     return jobByUser;
   }
 
+  async getAllJobPost() {
+    const allPosts = await this.jobPostRepository.find({
+      relations: ['jobSkills', 'jobCategory'],
+    });
+    return allPosts;
+  }
+
+  async findJobPostByUserId(userId: number) {
+    const getAllByUserId = await this.jobPostRepository.find({
```

Comment:
query builder

Reply:
I delete this function and use getJobPostByUser, and change that to query builder.